python/PiFinder/ui/catalog.py

```diff
@@ -73,6 +73,12 @@ class UICatalog(UIModule):
             "options": ["CANCEL", 5, 10, 15, 20],
             "callback": "push_near",
         },
+        "Push All Near": {
```

Comment:
I think these two option descriptions need some work..... but I don't know what to do to make them really intuitive with the limited text length.  Ideally they would be something like:

* Find nearby objects in current catalog
* Find nearby objects in all catalogs

I'll think about this a bit, but if you have any suggestions let me know :-)

Reply:
agreed, not sure what's better.
Near in Cat
Near in All
?

Reply:
Thank you!!  That's better than the current state, let's go with that for now!